.travis.yml

```diff
@@ -40,6 +42,12 @@ before_install:
   - bash scripts/travis_ci_prerequisites.sh
   - export PATH="$PATH:$HOME/.cargo/bin"
   - export PATH="$PATH:$HOME/protoc/bin"
+  - git clone https://github.com/pyenv/pyenv-virtualenv.git $(pyenv root)/plugins/pyenv-virtualenv
+  - echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.bash_profile
+  - source ~/.bash_profile
+  - pyenv virtualenv 3.7.1 tests
+  - pyenv activate tests
+  - pip3 install -r integration_tests/requirements.txt
```

Comment:
I am just curious that how about installing python3, python3-venv, python3-pip through addons?

```
addons:
  apt:
    packages:
      - curl
      - python3
      - python3-venv
      - python3-pip
  homebrew:
    package:
      - curl
      - python3 ......
```

Is it more verbose?

Reply:
`pyenv` and its plugin is more developed tool, and it's already installed in default Travis CI imgae.
Better to use it.

Reply:
Need to move these lines to Makefile?

Reply:
If this is travis specific thing, I think do not have to put them to Makefile, But if it is, we can move them to travis_ci_prerequisites.sh ;)

Reply:
As these packages and plugins are needed only in CI, will pour them into `travis_ci_prerequisites.sh`